.github/workflows/terraform_checks.yml

```diff
@@ -7,8 +7,6 @@ on:
       - synchronize
       - reopened
   push:

```

Comment:
I suspect this works fine 😊. From a quick look I came across the below, which might be clearer and easier to adapt later down the line (while behaving the same)

```
on:
  push:
    branches:
      - '*'
```

Reply:
Happy with either if you don't feel like it's worth trying the above